src/configure/clients/devOps/azureDevOpsClient.ts

```diff
@@ -56,47 +62,47 @@ export class AzureDevOpsClient {
                 "name": projectName,
                 "visibility": 0,
                 "capabilities": {
-                    "versioncontrol": {"sourceControlType": "Git" },
+                    "versioncontrol": { "sourceControlType": "Git" },
                     "processTemplate": { "templateTypeId": "adcc42ab-9882-485e-a3ed-7678f01f66bc" }
                 }
             },
             deserializationMapper: null,
             serializationMapper: null
         })
-        .then((operation) => {
-            if(operation.url) {
-                return this.monitorOperationStatus(operation.url);
-            }
-            else {
-                throw new Error(util.format(Messages.failedToCreateAzureDevOpsProject, operation.message));
-            }
-        });
+            .then((operation) => {
```

Comment:
this tab spacing is redundant

Reply:
This got changed with auto formatting setting from vscode. Keeping this change for now, will remove the setting from editor so it doesn't happen again.